test/boostTest.cpp

```diff
@@ -21,4 +21,8 @@
  */
 
 #define BOOST_TEST_MODULE EthereumTests
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
```

Comment:
No idea for windows. There are several boost includes which cause the same warnings for macos.